oot.sh

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/bash
+
+while getopts ':c:' OPT; do
+        case ${OPT} in
+        c)
+		case ${OPTARG} in
+		clang|gcc|linaro_gcc)
+			export _compiler=${OPTARG}
+			;;
+		*)
+			echo "Compiler '${OPTARG}' unknown or not implemented"
+			exit
+			;;
+		esac
+		;;
+        ?)
+                echo && echo "Incorrect option: ${OPT}"
+                exit
+                ;;
+        esac
```

Comment:
Github is sneakily hiding this detail (making the global problem even larger), but all these lines (+ line 4 & 5) are indented with 4 spaces while the rest uses tabs.

The other (`compile*.sh`) files use 4 spaces, would you mind making this consistent across the new file?

Reply:
Done.

Reply:
Ah when the argument is not recognized they pass it through `OPTARG` :slightly_smiling_face: